manager/src/main/java/com/redhat/service/bridge/manager/api/user/BridgesAPI.java

```diff
@@ -52,26 +61,57 @@
     @Inject
     JsonWebToken jwt;
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
+    })
+    @Operation(summary = "Get the list of ingresses", description = "Get the list of ingresses for the authenticated user.")
     @GET
     public Response getBridges(@Valid @BeanParam QueryInfo queryInfo) {
         System.out.println(jwt.getSubject());
         return Response.ok(ListResponse.fill(bridgesService
                 .getBridges(identityResolver.resolve(jwt), queryInfo), new BridgeListResponse(), bridgesService::toResponse)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Accepted.", responseCode = "202",
```

Comment:
```suggestion
            @APIResponse(description = "Accepted.", responseCode = "201",
```

Reply:
Ah, you are right. Actually I think 202 should be better since we are processing the creation asynchronously. wdyt @robpblake @sutaakar ? If you are ok, I'll change the api accordingly

Reply:
Fine for me to use HTTP 202

Reply:
done